src/components/Layer/Layer.tsx

```diff
@@ -1,74 +1,133 @@
+
 import * as React from 'react';
```

Comment:
It would be a good idea to demonstrate using a layer that targets a specific nexus.